desktop/src/reports/report-config-panel/report-config-panel.component.html

```
@@ -0,0 +1,150 @@
+    <div class="report-config__list-row">
+      <span class="report-config__pos">{{ level.index + 1 }}</span>
+      <span class="report-config__list-label">{{ level.label }}</span>
+      <app-button matButtonType="iconButton" icon="arrow_upward" tooltip="Move up" [disabled]="level.isFirst"
+        (clicked)="moveGroupBy(level.index, -1)" data-testid="report-grouping-up"></app-button>
+      <app-button matButtonType="iconButton" icon="arrow_downward" tooltip="Move down" [disabled]="level.isLast"
+        (clicked)="moveGroupBy(level.index, 1)" data-testid="report-grouping-down"></app-button>
+      <app-button matButtonType="iconButton" icon="close" color="warn" tooltip="Remove"
+        (clicked)="removeGroupBy(level.index)" data-testid="report-grouping-remove"></app-button>
+    </div>
+  }
+  @if (addableDimensions().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)